src/components/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+// Action
+const CHECK_STATUS = 'CHECK_STATUS ';
```

Comment:
- [ ] Please follow the duck pattern for values of each action so it will be like:
```
const CHECK_STATUS = bookstore/categories/CHECK_STATUS
```

Reply:
@gemmen29 I've implemented the changes. Thank you for the feedback to make my codebase better. Cheers!